### DISCUSSION TITLE
NextUi + NextJs 13

### DISCUSSION THREAD

**Discussion by stone89son** [heroui-inc/heroui#1155] [Feedback]
I followed the installation steps from your page "https://nextui.org/docs/guide/nextui-plus-nextjs," but it didn't work for me. I encountered numerous errors. Can you provide a step-by-step guide with examples? I believe many others may face the same issue
Example: https://github.com/nextui-org/nextui/issues/849

**Comment by jrgarciadev** [comment]
Please use v2, here's a template repo 

With app directory: https://github.com/nextui-org/next-app-template
Wit pages directory: https://github.com/nextui-org/next-pages-template

V2 documentation: https://nextui-docs-v2.vercel.app/
